crates/astria-sequencer/src/app.rs

```diff
@@ -102,14 +102,14 @@ const MAX_SEQUENCE_DATA_BYTES_PER_BLOCK: usize = 256_000;
 pub(crate) struct App {
     state: InterBlockState,
 
-    // set to true when `prepare_proposal` is called, indicating we are the proposer for this
-    // block. set to false when `process_proposal` is called, as it's called during the prevote
-    // phase for that block.
+    // The validator address in cometbft being used to sign votes.
     //
-    // if true, `process_proposal` is not executed, as this means we are the proposer of that
-    // block, and we have already executed the transactions for the block during
-    // `prepare_proposal`, and re-executing them would cause failure.
-    is_proposer: bool,
+    // Used to avoid executing a block in both `prepare_proposal` and `process_proposal`. It
+    // is set in `prepare_proposal` from information sent in from cometbft and can change.
+    // In `process_proposal` we check if we prepared the proposal, and if so, we clear the
+    // value and we skip re-execution of the block's transactions to avoid failures caused by
+    // re-execution.
+    signing_address: Option<account::Id>,
```

Comment:
```suggestion
   validator_address: Option<account::Id>,
```

kinda a nit but specifies that this is for being a validator, signing is a bit general imo